[2025-10-31T22:57:01.442+0000] {processor.py:161} INFO - Started process (PID=170) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T22:57:01.450+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T22:57:01.461+0000] {logging_mixin.py:188} INFO - [2025-10-31T22:57:01.457+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T22:57:01.618+0000] {logging_mixin.py:188} INFO - [2025-10-31T22:57:01.569+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T22:57:01.638+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T22:57:02.098+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 0.689 seconds
[2025-10-31T22:57:32.666+0000] {processor.py:161} INFO - Started process (PID=178) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T22:57:32.671+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T22:57:32.678+0000] {logging_mixin.py:188} INFO - [2025-10-31T22:57:32.677+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T22:57:32.778+0000] {logging_mixin.py:188} INFO - [2025-10-31T22:57:32.760+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T22:57:32.784+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T22:57:32.883+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 0.231 seconds
[2025-10-31T22:58:07.933+0000] {processor.py:161} INFO - Started process (PID=189) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T22:58:07.981+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T22:58:08.092+0000] {logging_mixin.py:188} INFO - [2025-10-31T22:58:08.074+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T22:58:09.981+0000] {logging_mixin.py:188} INFO - [2025-10-31T22:58:09.701+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T22:58:10.108+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T22:58:11.522+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 3.955 seconds
[2025-10-31T22:58:41.871+0000] {processor.py:161} INFO - Started process (PID=200) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T22:58:41.875+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T22:58:41.877+0000] {logging_mixin.py:188} INFO - [2025-10-31T22:58:41.877+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T22:58:41.920+0000] {logging_mixin.py:188} INFO - [2025-10-31T22:58:41.912+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T22:58:41.924+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T22:58:41.983+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 0.124 seconds
[2025-10-31T23:02:17.908+0000] {processor.py:161} INFO - Started process (PID=170) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:02:17.955+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:02:18.019+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:02:18.001+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:02:18.201+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:02:18.181+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:02:18.206+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:02:18.541+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 0.724 seconds
[2025-10-31T23:02:56.823+0000] {processor.py:161} INFO - Started process (PID=178) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:02:56.847+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:02:56.865+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:02:56.862+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:02:57.784+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:02:57.497+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:02:57.799+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:02:58.317+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 1.698 seconds
[2025-10-31T23:03:33.327+0000] {processor.py:161} INFO - Started process (PID=189) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:03:33.392+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:03:33.746+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:03:33.529+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:03:36.934+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:03:36.503+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:03:37.009+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:03:40.002+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 7.095 seconds
[2025-10-31T23:04:21.576+0000] {processor.py:161} INFO - Started process (PID=200) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:04:21.905+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:04:22.292+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:04:22.266+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:04:23.091+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:04:22.988+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:04:23.121+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:04:23.736+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 4.033 seconds
[2025-10-31T23:04:54.377+0000] {processor.py:161} INFO - Started process (PID=211) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:04:54.414+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:04:54.441+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:04:54.430+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:04:54.617+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:04:54.589+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:04:54.625+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:04:55.083+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 0.787 seconds
[2025-10-31T23:05:27.671+0000] {processor.py:161} INFO - Started process (PID=222) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:05:27.680+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:05:27.688+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:05:27.685+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:05:27.762+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:05:27.744+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:05:27.767+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:05:27.855+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 0.211 seconds
[2025-10-31T23:05:58.888+0000] {processor.py:161} INFO - Started process (PID=231) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:05:58.902+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:05:58.911+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:05:58.908+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:05:59.091+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:05:59.060+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:05:59.098+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:05:59.869+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 1.043 seconds
[2025-10-31T23:06:39.733+0000] {processor.py:161} INFO - Started process (PID=238) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:06:39.784+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:06:39.827+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:06:39.807+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:06:40.328+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:06:40.240+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:06:40.343+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:06:41.693+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 2.072 seconds
[2025-10-31T23:07:18.651+0000] {processor.py:161} INFO - Started process (PID=249) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:07:18.671+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:07:18.718+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:07:18.707+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:07:19.048+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:07:18.983+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:07:19.065+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:07:19.561+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 0.991 seconds
[2025-10-31T23:07:50.912+0000] {processor.py:161} INFO - Started process (PID=258) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:07:50.990+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:07:51.035+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:07:51.018+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:07:51.304+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:07:51.283+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:07:51.310+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:07:52.279+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 1.735 seconds
[2025-10-31T23:08:27.640+0000] {processor.py:161} INFO - Started process (PID=265) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:08:27.895+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:08:28.011+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:08:28.008+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:08:28.263+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:08:28.215+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:08:28.292+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:08:28.529+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 2.068 seconds
[2025-10-31T23:09:09.186+0000] {processor.py:161} INFO - Started process (PID=279) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:09:09.240+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:09:09.256+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:09:09.253+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:09:09.433+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:09:09.403+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:09:09.442+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:09:09.971+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 1.120 seconds
[2025-10-31T23:09:40.893+0000] {processor.py:161} INFO - Started process (PID=287) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:09:40.908+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:09:40.926+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:09:40.918+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:09:41.146+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:09:41.111+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:09:41.156+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:09:41.687+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 0.892 seconds
[2025-10-31T23:10:17.119+0000] {processor.py:161} INFO - Started process (PID=298) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:10:17.124+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:10:17.134+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:10:17.132+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:10:17.354+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:10:17.314+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:10:17.364+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:10:17.585+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 0.522 seconds
[2025-10-31T23:10:50.326+0000] {processor.py:161} INFO - Started process (PID=309) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:10:50.367+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:10:50.424+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:10:50.386+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:10:50.706+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:10:50.689+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:10:50.718+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:10:51.030+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 0.720 seconds
[2025-10-31T23:11:24.334+0000] {processor.py:161} INFO - Started process (PID=320) to work on /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:11:24.389+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/load_to_clickhouse.py for tasks to queue
[2025-10-31T23:11:24.530+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:11:24.511+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:11:26.420+0000] {logging_mixin.py:188} INFO - [2025-10-31T23:11:26.247+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/load_to_clickhouse.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/load_to_clickhouse.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2025-10-31T23:11:26.552+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/load_to_clickhouse.py
[2025-10-31T23:11:27.880+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/load_to_clickhouse.py took 3.727 seconds
